operator/controllers/actionsrunner/util/converters.go

```diff
@@ -114,7 +115,7 @@ func ToConfigMap(dotFiles *dot.Files, actionsRunner *inlocov1alpha1.ActionsRunne
 		return nil, err
 	}
 
-	configMap := corev1.ConfigMap{
+	configMapCredentials := corev1.ConfigMap{
```

Comment:
I would revert this change, the ConfigMap may hold other values in the future depending on actions/runner implementation.